### ISSUE TITLE
печать Z-отчета

### ISSUE THREAD

**Issue by Dafafgh** [VitalyKravtsov2016/OposWebkassa#79]
непонятно по какой команде сейчас снимается з-отчет? не привязан ли он к суткам, как в случае с токеном авторизации?

в логах реталикса 4 момента, где должен был быть снят з-отчет, но снятие отчета видно только в 23:52:14

00:14:21.950  ..........  Info......  TFPWebKassa.........  GetFiscalZNumber....  GetData(FPTR_GD_Z_REPORT) sData=336, Result=336
08:01:54.689  ..........  Info......  TFPWebKassa.........  GetFiscalZNumber....  GetData(FPTR_GD_Z_REPORT) sData=337, Result=337
08:08:44.024  ..........  Info......  TFPWebKassa.........  GetFiscalZNumber....  GetData(FPTR_GD_Z_REPORT) sData=337, Result=337

23:52:14.102  ..........  Info......  TFPWebKassa.........  GetFiscalZNumber....  GetData(FPTR_GD_Z_REPORT) sData=337, Result=337
[08.09.2024 23:52:14.134 ] [00013972] [DEBUG] Post: http://localhost:1331/api/zreport

[sssss.zip](https://github.com/user-attachments/files/17211766/sssss.zip)

**Comment by VitalyKravtsov2016** [comment]
Можно подробнее описать проблему?

**Comment by Dafafgh** [comment]
так, тут все нормально. я не нашел какой именно командой реталикс отправляет снятие з-отчета. но нашел проблему в реталиксе. в этих случаях реталикс не отправлял снятие з-отчета
